tests/test_iib_build_details_model.py

```diff
@@ -233,6 +234,7 @@ def test_from_dict_failure(
         batch=1,
         batch_annotations={"batch_annotations": 1},
         logs={},
+        deprecation_list=[],
```

Comment:
This move should not be in this code... you can update your first commit and put `deprecation_list` below `logs` directly so this step is not necessary

Reply:
After discussion with @lipoja  -> All commits will be squashed and merged so we don't have to pay attention to this comment.